### ISSUE TITLE
Casting to number and vice versa

### ISSUE THREAD

**Issue by wolfv** [mitsuhiko/minijinja#371]
I quickly looked and couldn't find a filter or function to cast from a string to a number or the other way around. 

E.g. `{{ ("1234" | int) > 1000 }}` could be useful or `{{ "check" if int("1234") > 1000 }}`. 

I was wondering if there are plans for this?

**Comment by mitsuhiko** [comment]
Yep. That's an omission that should be added. `int` exists in Jinja2.

**Comment by wolfv** [comment]
Awesome, thanks!
